bcda/auth/alpha_test.go

```diff
@@ -136,6 +136,10 @@ func (s *AlphaAuthPluginTestSuite) TestAccessToken() {
 	assert.Nil(s.T(), err)
 	assert.NotEmpty(s.T(), ts)
 	assert.Regexp(s.T(), regexp.MustCompile(`[^.\s]+\.[^.\s]+\.[^.\s]+`), ts)
+	t, _ := s.p.DecodeJWT(ts)
+	c := t.Claims.(*auth.CommonClaims)
+	assert.True(s.T(), c.ExpiresAt <= time.Now().Unix()+3600)
+	assert.False(s.T(), c.ExpiresAt > time.Now().Unix()+3600)
```

Comment:
With both assertions equivalent, shall we just keep the first?

Reply:
yeah, i think so!